### PR TITLE
Adding Terms and Conditions to charm metadata.

### PR DESCRIPTION
**Pull request by alesstimec**

### CODE REVIEW

File: internal/test-charm-repo/quantal/terms/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: terms
+summary: "Sample charm with terms and conditions"
+description: |
+        That's a boring charm that requires certain terms.
+terms: ["term1", "term2"]
```
Comment (mhilton) [inline internal/test-charm-repo/quantal/terms/metadata.yaml]: should this have a newline?

**Comment by mhilton** [comment]
LGTM

**Comment by urosj** [comment]
:+1:

File: internal/test-charm-repo/quantal/terms/metadata.yaml (hunk 1 above)
Comment (alesstimec) [inline internal/test-charm-repo/quantal/terms/metadata.yaml]: done.

**Comment by urosj** [comment]
@alesstimec good catch, thanks.

**Comment by alesstimec** [comment]
$$merge$$

**Pull request merged**